docs/API.md

```diff
@@ -1073,6 +1073,20 @@ Example: `GET trusty/juju-gui-3/meta/charm-config`
 }
 ```
 
+#### GET *id*/meta/terms
+
+The `meta/terms` path returns a list of terms and angreements the user must
+agree to in order to obtain the archive of the given charm or bundle id.
```

Comment:
Perhaps document that this endpoint comes directly from the terms field in the charm or bundle metadata?